loopy/kernel/__init__.py

```diff
@@ -1638,25 +1603,45 @@ def __ne__(self, other):
 
     # }}}
 
+    def get_copy_kwargs(self, **kwargs):
+        if "iname_to_tags" in kwargs:
+            assert "inames" not in kwargs
```

Comment:
Input checking shouldn't be done in `assert`.

Reply:
Thanks: 97e5699.